django/db/migrations/migration.py

```diff
@@ -41,6 +41,17 @@ class Migration(object):
     # are not applied.
     replaces = []
 
+    # Initial migration, should be silently skipped on --fake-initial if table
```

Comment:
Is this an initial migration? Initial migrations are skipped on --fake-initial...